src/main/java/com/redhat/jenkins/plugins/ci/GlobalCIConfiguration.java

```diff
@@ -82,6 +87,7 @@ public GlobalCIConfiguration(List<JMSMessagingProvider> configs) {
         this.configs = configs;
     }
 
+    @DataBoundConstructor
```

Comment:
Not sure if ever used, but `@DataBoundSetter` is reported not to work without `@DataBoundConstructor`.